include/MainWindow.h

```diff
@@ -224,6 +225,8 @@ private slots:
 	ToolButton * m_metronomeToggle;
 
 	SessionState m_session;
+	
+	bool maximised;
```

Comment:
This variable is only ever assigned, never read?

Reply:
Oh no, I left the reading out from my debug phase xD
Let me fix that real quick

Reply:
Yep, now its being read to restore it to its previous state (maximised or normal).

Reply:
Ok. Since I thought it would get removed I didn't mention this, but for consistency's sake it should probably be renamed to maximized 😅. Sorry for making a bunch of separate comments instead of a single review pass 😬.

Reply:
> renamed to maximized

Lol sure I'll do that

Reply:
Did that